node/config/types.go

```diff
@@ -261,34 +261,34 @@ type StorageConfig struct {
 	ParallelFetchLimit int
 }
 
-type PieceDirectoryCouchbaseBucketConfig struct {
+type LocalIndexDirectoryCouchbaseBucketConfig struct {
 	// Bucket setting RAMQuotaMB
 	RAMQuotaMB uint64
 }
 
-type PieceDirectoryCouchbaseConfig struct {
+type LocalIndexDirectoryCouchbaseConfig struct {
```

Comment:
We have to at least contact @TippyFlitsUK when we merge this, as we'll have to update config for all Boost nodes that are already running Piece Directory. I know we haven't released it yet, but there are at least 3 Boost nodes that are already on it.

Reply:
I don't think @TippyFlitsUK is actually running piece directory in production yet - he just ran the migration script